pkg/cloudprovider/providers/aws/volumes.go

```diff
@@ -51,6 +52,8 @@ func mapVolumeSpecToAwsID(name KubernetesVolumeID) (awsVolumeID, error) {
 	s := string(name)
 
 	if !strings.HasPrefix(s, "aws://") {
+		// Assume a bare aws volume id (vol-1234...)
+		// Build a URL with an empty host (AZ)
 		s = "aws://" + "" + "/" + s
```

Comment:
It is supposed to be a placeholder to make it clear that we're using an empty host.  Maybe I should just remove it given that everyone else seems to hate it...